admin/plugins.php

```diff
@@ -390,3 +458,87 @@ function perflab_render_plugin_card( array $plugin_data ) {
 	</div>
 	<?php
 }
+
+// WordPress AJAX action to handle the button click event.
+add_action( 'wp_ajax_perflab_install_activate_standalone_plugins', 'perflab_install_activate_standalone_plugins_callback' );
+add_action( 'wp_ajax_nopriv_perflab_install_activate_standalone_plugins', 'perflab_install_activate_standalone_plugins_callback' );
```

Comment:
I don't think this needs to be available for unprivileged users? They wouldn't (and shouldn't) be able to install or activate plugins anyway.